config/routes.rb

```diff
@@ -110,6 +110,8 @@
       end
     end
 
+    resource :consent, only: [:create]
```

Comment:
what about nesting this under `/members/`. You're even passing the member id for the endpoint.